features/step_definitions/civil_journey_steps.rb

```diff
@@ -543,8 +543,8 @@
   expect(page).to have_content("Enter your client's correspondence address")
 end
 
-Then('I am on the client use online banking page') do
-  expect(page).to have_content('Check if you can continue using this service')
+Then('I am on the provider open banking consent page') do
```

Comment:
Is this still used? I couldn't see it on the new examples

Reply:
Shouldn't be , thank you. I removed it